src/Models/ForeignRelationship.php

```diff
@@ -62,7 +62,7 @@ class ForeignRelationship implements JsonWriter
     /**
      * Instance of the foreign model.
      *
-     * @var Illuminate\Database\Eloquent\Model
+     * @var Model
```

Comment:
I cleaned up these type annotations mainly in self-defence - PHPStorm lit up with the highlights.